LiskHQ/lisk-sdk
# Change lisk-blocks to lisk-chain - Closes #4766

### What was the problem?
Refer https://github.com/LiskHQ/lisk-sdk/issues/4766 for motivation
### How did I solve it?
- Rename `lisk-blocks` library to `lisk-chain`
- Update framework reference and all the tests
### How to manually test it?
- Clean install
- Run test app
- Build should be green
### Review checklist

- [ ] The PR resolves #4766
